ccdImports/configFiles/CCD_Probate_Backoffice/CaseEvent.json

```diff
@@ -134,62 +134,63 @@
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "boWithdrawApplicationForCaseMatchingIssueGrant", "Name": "Withdraw application", "Description": "Withdraw application", "DisplayOrder": 188, "PreConditionState(s)": "BOCaseMatchingIssueGrant", "PostConditionState": "BOCaseClosed", "CallBackURLAboutToSubmitEvent": "http://${CCD_DEF_CASE_SERVICE_BASE_URL}/tasklist/update", "SecurityClassification": "public", "ShowSummary": "N", "ShowEventNotes": "Y"},
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "boWithdrawApplicationForCaseMatchingExamining", "Name": "Withdraw application", "Description": "Withdraw application", "DisplayOrder": 189, "PreConditionState(s)": "BOCaseMatchingExamining", "PostConditionState": "BOCaseClosed", "CallBackURLAboutToSubmitEvent": "http://${CCD_DEF_CASE_SERVICE_BASE_URL}/tasklist/update", "SecurityClassification": "public", "ShowSummary": "N", "ShowEventNotes": "Y"},
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "boWithdrawForRedeclarationSOT", "Name": "Withdraw redeclaration", "Description": "Withdraw redeclaration", "DisplayOrder": 190, "PreConditionState(s)": "BOCaseStoppedAwaitRedec", "PostConditionState": "BOCaseStopped", "CallBackURLAboutToSubmitEvent": "http://${CCD_DEF_CASE_SERVICE_BASE_URL}/tasklist/update", "SecurityClassification": "Public", "ShowSummary": "N", "ShowEventNotes": "Y"},
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "citizenHubResponse", "Name": "Citizen hub response", "Description": "Citizen hub response", "DisplayOrder": 191, "PreConditionState(s)": "BOCaseStopped;Dormant", "PostConditionState": "*", "CallBackURLAboutToSubmitEvent": "http://${CCD_DEF_CASE_SERVICE_BASE_URL}/document/citizenHubResponse","SecurityClassification": "Public", "ShowSummary": "N", "ShowEventNotes": "Y"},
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "citizenHubResponseDraft", "Name": "Citizen hub response draft", "Description": "Citizen hub response draft", "DisplayOrder": 192, "PreConditionState(s)": "BOCaseStopped;Dormant", "PostConditionState": "*", "CallBackURLAboutToSubmitEvent": "http://${CCD_DEF_CASE_SERVICE_BASE_URL}/document/citizenHubResponse","SecurityClassification": "Public", "ShowSummary": "N", "ShowEventNotes": "Y"}
-]
\ No newline at end of file
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "boResolveCaveatStop", "Name": "Resolve Caveat Stop", "Description": "Resolve Caveat Stop", "DisplayOrder": 135, "PreConditionState(s)": "BOCaveatPermenant", "PostConditionState": "*", "SecurityClassification": "Public", "CallBackURLAboutToSubmitEvent": "http://${CCD_DEF_CASE_SERVICE_BASE_URL}/case/resolveCaveatStopState","ShowSummary": "N", "ShowEventNotes": "Y"},
```

Comment:
i'm going to assume that the diff here is just  everything moving down by one but i don't know if we have a good way to actually verify that nothing else has changed